src/index.css

```diff
@@ -12,6 +12,17 @@
   --accent-purple: #8b5cf6;
   --accent-green: #10b981;
   --accent-yellow: #f59e0b;
+
+--animate-fade-in: fade-in 0.5s ease forwards;
+
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
 }
```

Comment:
small format issue here, maybe we should add prettier to execute on `.css` files in `.vscode` workspace configuration file too